packit/config/package_config.py

```diff
@@ -150,6 +150,11 @@ def get_from_dict(
             else:
                 raise PackitConfigException("Spec file was not found!")
 
+        if not Path(raw_dict["specfile_path"]).is_file():
+            raise PackitConfigException(
+                f"Spec file not found under {raw_dict['specfile_path']}"
+            )
+
```

Comment:
Could we move this code to the validate command and change it to log a message which would say:

    The spec file you defined {specfile_path} is not present in the repository. If it's being generated dynamically, you can verify the functionality by running `packit srpm` to create an SRPM from the current checkout. If it's not being generated, please make sure the path is correct and the file is present.